control/dns_control.go

```diff
@@ -28,6 +31,9 @@ import (
 	"github.com/daeuniverse/outbound/netproxy"
 	"github.com/daeuniverse/outbound/pkg/fastrand"
 	"github.com/daeuniverse/outbound/pool"
+	tc "github.com/daeuniverse/outbound/protocol/tuic/common"
+	"github.com/daeuniverse/quic-go"
+	"github.com/daeuniverse/quic-go/http3"
```

Comment:
imported and not used

Reply:
No, they used here https://github.com/EkkoG/dae/blob/08eceb2dd22aca07ed3354645c33faa713556766/control/dns_control.go#L644-L660